agent/src/agent/pipeline/pipeline.py

```diff
@@ -195,30 +195,45 @@ def values(self) -> dict:
         return self.config.get('values', {})
 
     @property
-    def value_names(self) -> list:
-        return list(self.config.get('values', {}).keys())
-
-    @property
-    def values_paths(self):
-        return [self._get_property_path(value) for value in self.value_names]
+    def value_paths(self) -> list:
+        return [self._get_property_path(value) for value in list(self.values.keys())]
 
     @property
     def target_types(self) -> list:
         if self.source.type == source.TYPE_INFLUX:
-            return [self.config.get('target_type', 'gauge')] * len(self.value_names)
-        return list(self.config['values'].values())
+            return [self.config.get('target_type', 'gauge')] * len(self.value_paths)
+        return list(self.values.values())
+
+    @property
+    def measurement_paths_with_names(self) -> dict:
```

Comment:
why we need this?

Reply:
well, for convenience. With this we don't need to use for loops with indices, we can use this method